.travis.yml

```diff
@@ -1,34 +1,43 @@
+# TravisCI configuration for php-annotations/php-annotations
+
 language: php
+os:
+  - "linux"
+dist: "xenial"
+
+php:
```

Comment:
1. why you've rearranged the sections
2. why you've added double quotes around arguments (everything worked without this before)

Depending on the answers I might ask you to revert the above-mentioned changes.

Reply:
Hello!

1. now it is in chronological order, as it happens during execution
2. if you always quote YAML strings you don't have to decide one-by-one whether it is necessary (e.g. floats like 7.4)

Reply:
actually all string should be quoted but I wasn't daring...

Reply:
Massive changes are best to discuss before doing them.

IMO adding double quotes all over the place and rearranging stuff without asking if that is ever needed is both non-productive from the developer and a code-reviewer side.

Please try to revert above mentioned changes and we'll see if what's left can be merged.

Reply:
Please see #134